metrics/session.go

```diff
@@ -151,6 +151,14 @@ var (
 			Help:      "Bucketed histogram of different states of a transaction.",
 			Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 29), // 0.5ms ~ 1.5days
 		}, []string{LblType, LblHasLock})
+	LazyPessimisticUniqueCheckSetCount = prometheus.NewCounter(
```

Comment:
why do we need add this metric?

Reply:
Mainly for telemetry. I chose to count the setting of the variable as an indicator of feature usage.